sys/linux/bpf_386.const

```diff
@@ -140,7 +140,7 @@ BPF_XADD0 = 6
 BPF_XOR0 = 10
 BTF_MAGIC = 60319
 BTF_VERSION = 1
-__BPF_FUNC_MAX_ID = 91
+__BPF_FUNC_MAX_ID = 84
```

Comment:
You are using an old kernel checkout. We always generate consts on tip of upstream tree.